jekyll/_cci2/monitoring.md

```diff
@@ -75,19 +75,25 @@ Enter your DataDog API Key.
 
 After saving you can *verify* that metrics are forwarding by going to the DataDog metrics summary.
 
-#### Custom Metrics *(beta)*
+#### Custom Metrics
 
-A beta feature is to configure Telegraf by going to Replicated Admin > Settings > Custom Metrics and enabling the custom option, for example `https://example.com:8800/settings#custom_metrics`.
-
-![Custom](  {{ site.baseurl }}/assets/img/docs/metrics_custom1.png)
+Custom Metrics via Telegraf configuration file is an alternative to the predefined CloudWatch and Datadog metric sections above. It can also be used instead of those sections for more fine grained control.
 
 ##### Configuration
 
 Configuration options are based on Telegraf's documented output plugins. See their documentation [here](https://github.com/influxdata/telegraf#output-plugins).
 
-For example, if you would like to use the InfluxDB Output Plugin the configuration would look like the following.
-
-![Custom](  {{ site.baseurl }}/assets/img/docs/metrics_custom2.png)
+For example, if you would like to use the InfluxDB Output Plugin you would need to follow these steps;
+1. SSH into the Servics Machine
+2. cd `/etc/circleconfig/telegraf/influxdb.conf
+3. Adding the desired ouputs, e.g. ```
```

Comment:
change e.g. to 'for example' and follwo it with a line break for correct formatting of the code block